distribution/artifact/rules.bzl

```diff
@@ -0,0 +1,10 @@
+load("@graknlabs_bazel_distribution//artifact:rules.bzl", "artifact_file")
+
+def native_artifact_files(name, artifact_name, **kwargs):
+    for platform in ["mac", "linux"]:
+        artifact_file(
+            name = name + "_" + platform,
+            # Can't use .format() because the result string will still have the unresolved parameter {version}
+            artifact_name = artifact_name.replace("{platform}", platform),
+            **kwargs,
+        )
```

Comment:
New Bazel macro to pull in both the Mac and Linux artifacts into the WORKSPACE of a client